### PR TITLE
HTML processing

### PR DESCRIPTION
**Pull request by ol0lll**

### CODE REVIEW

**Comment by chrfritsch** [comment]
With Firefox latest, I have to press Ctrl-V two times to trigger the paste event.

**Comment by chrfritsch** [comment]
Pasting two times a different text, creates two paragraphs with the same text

**Comment by chrfritsch** [comment]
With split method "oEmbed" enabled, I get some empty paragraphs by pasting something like

```
essen. Hunger kennen die meisten Menschen in Deutschland nicht mehr.

https://www.youtube.com/watch?v=4ksufBoeZm8

Das hat sich auch auf unsere Körper ausgewirkt: Laut der Ernährungsmedizinerin Dr. Petra

File: js/paragraphs-paste.js [hunk 1]
```diff
@@ -3,66 +3,97 @@
...
+    var targetElement = document.querySelector('[data-drupal-selector="' + event.currentTarget.dataset.paragraphsPasteTarget.replace(/action$/, 'content-value') + '"]');
+    var editor = CKEDITOR.instances[targetElement.id];
 
-    // Get pasted data via clipboard API.
-    clipboardData = event.originalEvent.clipboardData || window.clipboardData;
-    pastedData = JSON.stringify(clipboardData.getData('Text'));
+    editor.focus();
+    var editableElem = editor.editable().$;
+    // Reset editor contents.
+    editableElem.innerHTML = '';
```
Comment (chrfritsch) [inline js/paragraphs-paste.js:24]: Looks like this is not needed

**Review by chrfritsch — CHANGES_REQUESTED** [review]

File: js/paragraphs-paste.js (hunk 1 above)
Comment (ol0lll) [reply js/paragraphs-paste.js:24]: I think we need that, when I paste a second time I get the content from before additionally to what I actually pasted

Comment (ol0lll) [reply js/paragraphs-paste.js:24]: Also if someone was to type in the editor while in focus it would be better to clear it.

Comment (chrfritsch) [reply js/paragraphs-paste.js:24]: With the form reset I did?

**Review by chrfritsch — APPROVED** [review]

**Comment by ol0lll** [comment]
Created follow-up for 
* Additional empty paragraphs created
https://www.drupal.org/project/paragraphs_paste/issues/3129870
* Format content according to target field type
https://www.drupal.org/project/paragraphs_paste/issues/3129875

**Pull request merged**